docs/lib/datastore/fragments/ios/data-access/query-sort-multiple-snippet.md

```diff
@@ -0,0 +1,37 @@
+<amplify-block-switcher>
+
+<amplify-block name="Listener (iOS 11+)">
+
+```swift
+let p = Post.keys
+Amplify.DataStore.query(Post.self, sort: .by(.ascending(Post.keys.rating),
+                                             .descending(Post.keys.title))) {
+    switch $0 {
+    case .success(let result):
+        print("Posts: \(result)")
+    case .failure(let error):
+        print("Error listing posts - \(error.localizedDescription)")
+    }
+}
+```
+
+</amplify-block>
+
+<amplify-block name="Combine (iOS 13+)">
+
+```swift
+let p = Post.keys
+let sink = Amplify.DataStore.query(Post.self, sort: .by(.ascending(Post.keys.rating),
```

Comment:
i haven't looked at the new Combine docs, what is the variable name assigned by those? Is there a better naming like `let subscriber`?

Reply:
Actually, I just wanted to align with Tim's docs. So I simply copied his code snippet and replace with `sort`. We can ask in the team of the variable naming I guess?

Reply:
if there is consistency then that should be fine